src/boot/efi/secure-boot.c

```diff
@@ -33,3 +35,86 @@ SecureBootMode secure_boot_mode(void) {
 #ifdef SBAT_DISTRO
 static const char sbat[] _used_ _section_(".sbat") = SBAT_SECTION_TEXT;
 #endif
+
+EFI_STATUS secure_boot_enroll_at(EFI_FILE *root_dir, const char16_t *path) {
+        assert(root_dir);
+        assert(path);
+
+        EFI_STATUS err = EFI_SUCCESS;
+
+        clear_screen(COLOR_NORMAL);
+
+        Print(L"Enrolling secure boot keys from directory: \\loader\\keys\\%s\n"
+              L"Warning: Enrolling custom Secure Boot keys might soft-brick your machine!\n",
+              path);
+
+        UINT32 timeout_sec = 15;
```

Comment:
(We've switched to standard types where it makes sense.)
```suggestion
        unsigned timeout_sec = 15;
```